orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/WaitForDownInstanceHealthTaskSpec.groovy

```diff
@@ -35,13 +35,13 @@ class WaitForDownInstanceHealthTaskSpec extends Specification {
     ["Discovery"]                 || [health: [[type: "Discovery", state: "Down"]]]                                                                      || true
     ["Discovery"]                 || [health: [[type: "Discovery", state: "OutOfService"]]]                                                              || true
     ["Discovery", "Other"]        || [health: [[type: "Other", state: "Down"]]]                                                                          || true
-    ["Amazon"]                    || [health: []]                                                                                                        || false
```

Comment:
Talked about it offline ... `Amazon` is no different than `LoadBalancer` for the purpose of this test.